nomad/provider_test.go

```diff
@@ -29,16 +30,38 @@ func TestProvider(t *testing.T) {
 
 var testProvider *schema.Provider
 var testProviders map[string]terraform.ResourceProvider
+var managementToken string
 
 func init() {
 	testProvider = Provider().(*schema.Provider)
 	testProviders = map[string]terraform.ResourceProvider{
 		"nomad": testProvider,
 	}
+
+	if v := os.Getenv("TF_ACC"); v == "1" {
```

Comment:
nit: [`resource.TestEnvVar`](https://godoc.org/github.com/hashicorp/terraform/helper/resource#TestEnvVar) can be used to avoid hardcoding this.